.github/workflows/lighthouse-prod.yml

```diff
@@ -0,0 +1,25 @@
+name: Lighthouse CI PROD
+
+on: workflow_dispatch
```

Comment:
setup as manual dispatch for testing, but can change to a CRON https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events